include/sensors.h

```diff
@@ -94,6 +98,28 @@ class Sensors
     return true;
   }
 
+  static const float BARO_MAX_CHANGE_RATE;
+  static const float BARO_SAMPLE_RATE;
+  static const float DIFF_MAX_CHANGE_RATE;
+  static const float DIFF_SAMPLE_RATE;
+  static const float SONAR_MAX_CHANGE_RATE;
+  static const float SONAR_SAMPLE_RATE;
+  static const int SENSOR_CAL_DELAY_CYCLES;
+  static const int SENSOR_CAL_CYCLES;
+  static const float BARO_MAX_CALIBRATION_VARIANCE;
+  static const float DIFF_PRESSURE_MAX_CALIBRATION_VARIANCE;
```

Comment:
Hey @drinkdhmo,  So, I'm wondering why have these `const`s.  Is it just for clarity?  Should they be parameters?